### PR TITLE
Fix tests after metadata update

### PR DESCRIPTION
**Pull request by imbstack**
The changes in #11 made these tests fail in travis consistently. This fixes it. Just merge if the tests pass!

### CODE REVIEW

**Comment by jonasfj** [comment]
I think @elenasolomon introduced this... and from looking at the code I would expect it to work.
So maybe we have a bug in the tests, or a bug in the code. I'll leave that to @elenasolomon.

**Comment by imbstack** [comment]
The issue is that the queue gets "created" in a test and the changes in #11 changed the metadata on that queue. Afaict, the way the test was originally written and what it intended to do was to never recreate that queue, and only use it as it already existed. Now that the metadata changed, the test will always fail, in particular because it changes the metadata back in the middle of the tests. Options other than this change include always deleting and recreating the queue at the start of every suite. This comes with the caveat that deleting and recreating queues in azure cannot happen too frequently. I _think_ you're only allowed to delete every 30 seconds or something along those lines?

**Comment by imbstack** [comment]
I roped you in here, @jonasfj, because I believe you wrote the original test and so perhaps can speak to the motivation of reusing the table somewhat?

File: test/queue_test.js [hunk 1]
```diff
@@ -104,7 +104,8 @@ suite("Queue", function() {
     // Don't actually want to change the meta-data as it would affect the
     // createQueue test case...
     return queue.setMetadata(queueName, {
-      purpose:  'testing'
+      purpose:  'testing',
+      applicationName: 'fast-azure-storage',
```
Comment (jonasfj) [inline test/queue_test.js]: Ah, okay... Can we include the spaces here too then? We want to include them because that's part of the test case :)

**Pull request merged**